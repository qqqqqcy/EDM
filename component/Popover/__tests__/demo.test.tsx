import webDemoTest from '@tests/shared/demoTest';
import ReactDOM from 'react-dom';
ReactDOM.createPortal = jest.fn().mockReturnValue(null);
webDemoTest('Popover');
