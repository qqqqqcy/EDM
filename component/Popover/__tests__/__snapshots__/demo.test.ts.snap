// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Popover correctly 1`] = `
<div>
  <button
    class="edm-button edm-button-small edm-button-default edm-button-radius"
    role="button"
  >
    test
  </button>
</div>
`;
